Makefile

```diff
@@ -77,7 +77,7 @@ test-docker-py: build
 	$(DOCKER_RUN_DOCKER) hack/make.sh binary test-docker-py
 
 validate: build
-	$(DOCKER_RUN_DOCKER) hack/make.sh validate-gofmt validate-dco validate-toml
+	$(DOCKER_RUN_DOCKER) hack/make.sh validate-gofmt validate-vet validate-dco validate-toml
```

Comment:
this is a nit, but I would prefer the order `validate-dco validate-gofmt validate-toml validate-vet` so alphabetical, but I relize it was already like this and not your change